commands/registry.go

```diff
@@ -140,54 +146,57 @@ func Repository() *Command {
 			Use:     "repository",
 			Aliases: []string{"repo", "r"},
 			Short:   "Display commands for working with repositories in a container registry",
-			Long:    "The subcommands of `doctl registry repository` help you command actions related to a repository.",
+			Long:    "The subcommands of `doctl registry repository` allow you to manage various properties of your repository.",
 		},
 	}
 
-	listRepositoriesDesc := `This command retrieves information about repositories in a registry, including:
+	listRepositoriesDesc := `Retrieves information about repositories in a registry, including:
   - The repository name
-  - The latest tag of the repository
+  - The latest tag for the repository
   - The compressed size for the latest tag
   - The manifest digest for the latest tag
   - The last updated timestamp
 `
-	CmdBuilder(
+	cmdListRepositories := CmdBuilder(
 		cmd,
 		RunListRepositories, "list",
 		"List repositories for a container registry", listRepositoriesDesc,
 		Writer, aliasOpt("ls"), displayerType(&displayers.Repository{}),
 		hiddenCmd(),
 	)
+	cmdListRepositories.Example = `The following example lists repositories in a registry named ` + "`" + `example-registry` + "`" + ` and uses the ` + "`" + `--format` + "`" + ` flag to return only the digest and update time of each repository: doctl registry repository list example-registry --format Digest,UpdatedAt`
```

Comment:
Same here:

```
$ doctl registry repository list asb --format Digest,UpdatedAt
Error: unknown column "Digest"
```

Only `Name`, `LatestTag`, `TagCount`, and `UpdatedAt` are valid here.